### PR TITLE
Finalize look of the popup survey.

### PR DESCRIPTION
**Pull request by ericawright**
r? @bryanbell
screen shot 2018-08-23 at 2 13 29 pm

Fixes: #12.
Fixes: #19.

### CODE REVIEW

File: src/privileged/trackers/framescript.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-/* global XPCOMUtils, sendAsyncMessage, docShell */
+/* global ChromeUtils, sendAsyncMessage, docShell */
 const {classes: Cc, interfaces: Ci} = Components;
 const trackerListener = {
-  QueryInterface: XPCOMUtils.generateQI(["nsIWebProgressListener", "nsISupportsWeakReference"]),
+  QueryInterface: ChromeUtils.generateQI(["nsIWebProgressListener", "nsISupportsWeakReference"]),
```
Comment (ericawright) [inline src/privileged/trackers/framescript.js]: this can be removed once #61 is merged

File: src/privileged/trackers/api.js [hunk 2]
```diff
@@ -6,8 +6,6 @@ ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
 
 const { EventManager, EventEmitter } = ExtensionCommon;
 
-const {Management: {global: {tabTracker}}} = ChromeUtils.import("resource://gre/modules/Extension.jsm", {});
```
Comment (ericawright) [inline src/privileged/trackers/api.js]: oddly, everything under `privileged` seems to share a namespace, so I was getting errors that this was redefined. I don't like it though

File: src/privileged/notificationBar/api.js [hunk 3]
```diff
@@ -49,38 +49,47 @@ class NotificationBarEventEmitter extends EventEmitter {
...
     ];
 
-    const populatePanel = (event) => {
-      // console.log("EVENT HAPPENED", event);
-      // Can edit the contents of the panel here if we want variations per branch
-      // if (event !== "shown") {
-      //   return;
-      // }
-    };
+    // option persistant is described as: "the notification will always
```
Comment (johannhof) [inline src/privileged/notificationBar/api.js]: nit: persistent

File: src/privileged/notificationBar/api.js [hunk 4]
```diff
@@ -49,38 +49,47 @@ class NotificationBarEventEmitter extends EventEmitter {
...
+    //                    persist even across tab and app changes (but not across
+    //                    location changes), until the user accepts or rejects
+    //                    the request. The notification will never be implicitly
+    //                    dismissed."
+    // This is not the case, it does not persist across tab changes
```
Comment (johannhof) [inline src/privileged/notificationBar/api.js]: I think what's meant is that the notification will reappear after leaving the tab and coming back, I would happily take a patch to m-c that improves this documentation :)

File: src/privileged/notificationBar/api.js [hunk 5]
```diff
@@ -49,38 +49,47 @@ class NotificationBarEventEmitter extends EventEmitter {
...
+    //                    addon name."
+    // It is bold, but not used in a header, we're working with it anyway.
+    const options = {
+      hideClose: true,
+      persistent: true,
+      autofocus: true,
+      name: "Firefox Survey: ",
+      popupIconURL: "chrome://branding/content/icon64.png",
+    };
+    recentWindow.PopupNotifications.show(browser, "fast-block-notification", "<> Did you reload this page to resolve loading issues?", null, primaryAction, secondaryActions, options);
```
Comment (johannhof) [inline src/privileged/notificationBar/api.js]: I feel like we should open a (backlog) issue to add translation files for this. There are examples from the TRR Study (for using WebExtension JSON locale files) and the Fx Monitor Study (for using .properties files) on how to do this.

File: src/privileged/trackers/api.js (hunk 2 above)
Comment (johannhof) [reply src/privileged/trackers/api.js]: See my comment in #71, I'm hesitant to remove this entirely...

File: src/privileged/notificationBar/api.js (hunk 5 above)
Comment (ericawright) [reply src/privileged/notificationBar/api.js]: Yes, but aren't we only testing in EN locales? I don't think we do shield studies outside of EN.

**Pull request merged**